README.md

```diff
@@ -157,10 +157,33 @@ ansible-playbook build-test-and-push.yml                           \
     --user       $user
 ```
 
-### Creating the database
+### Deploying to the cluster
+
+``` bash
+cd concent-deployment/concent-builder/
+ansible-playbook deploy.yml                                        \
+    --extra-vars cluster=$cluster                                  \
+    --inventory  ../../concent-deployment-values/ansible_inventory \
+    --user       $user
+```
+
+### Create or delete the database
+This is a separate step that require access to the whole Google Cloud project to modify CloudSQL database.
+In normal scenario, we don't needed to create or delete database. Normal cluster deployment only involves a reset or a migration.
+
+```bash
+cd concent-deployment/cloud/
+ansible-playbook create-databases.yml                              \
```

Comment:
if there will be only one database, there could be singular instead of plural in playbook names

Reply:
There are two. `storage` and `control`.